packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -226,7 +226,7 @@ export function importAnalysisPlugin(config: ResolvedConfig): Plugin {
         return null
       }
 
-      const start = performance.now()
+      const msAtStart = debug && performance.now()
```

Comment:
I think we can do this instead so it's consistent with the others
```suggestion
      const msAtStart = debug ? performance.now() : 0
```

https://github.com/vitejs/vite/blob/bdb826ca0a021c71d6a43d1f6c81fd3e906eea56/packages/vite/src/node/server/pluginContainer.ts#L689

https://github.com/vitejs/vite/blob/bdb826ca0a021c71d6a43d1f6c81fd3e906eea56/packages/vite/src/node/server/pluginContainer.ts#L785

https://github.com/vitejs/vite/blob/bdb826ca0a021c71d6a43d1f6c81fd3e906eea56/packages/vite/src/node/server/transformRequest.ts#L204

Reply:
Ah, yes, changed! Thanks!